script/tool/lib/src/test_command.dart

```diff
@@ -32,72 +32,64 @@ class TestCommand extends PluginCommand {
       'This command requires "flutter" to be in your path.';
 
   @override
-  Future<void> run() async {
-    final List<String> failingPackages = <String>[];
-    await for (final Directory packageDir in getPackages()) {
-      final String packageName =
-          p.relative(packageDir.path, from: packagesDir.path);
-      if (!packageDir.childDirectory('test').existsSync()) {
-        print('SKIPPING $packageName - no test subdirectory');
-        continue;
-      }
+  Future<PackageResult> runForPackage(Directory package) async {
+    if (!package.childDirectory('test').existsSync()) {
+      return PackageResult.skip('No test/ directory.');
+    }
 
-      print('RUNNING $packageName tests...');
```

Comment:
There's a lot of missing "reporting". Is that done by the PackageLoopingCommand now? (not that I'm going to miss it, just making sure that the output log stays useful)

Reply:
Standardized output for starting each package's run is part of the base command, yes.